### ISSUE TITLE
Don't simplify expression before plotting them?

### ISSUE THREAD

**Issue by Ecco** [numworks/epsilon#366]
It is expected that `(x^2-3x)/x` gets simplified to `x-3`, even though this is false if x=0.

That being said, this raises an issue when plotting `(x^2-3x)/x`, because this function should be evaluated to `undef` in zero.

**Comment by adriweb** [comment]
Another potential idea: first determine undefined points (symbolically or not, I’m not sure) and plot the simplified equation except on those determined points where it’s just undef.

**Comment by adriweb** [comment]
What matters more however is the table/values view, as values need to be calculated from the original expressions. Plots with undef values on specific points are basically invisible as you’d have to zoom in infinitely to see them...
At least that’s the behavior I’m seeing on other calcs ; that said, why are functions simplified? Faster plotting in case the original function is complicated but the simplified one is much simpler?

**Comment by Ecco** [comment]
> Plots with undef values on specific points are basically invisible

That's true in theory, but graphing calculators are discrete machines by design. Which means you can very well scroll to an even zero using the dpad :)

> Faster plotting in case the original function is complicated but the simplified one is much simpler?

That was the idea. As always, premature optimization is the root of all evil :)
